### DISCUSSION TITLE
#Belived

### DISCUSSION THREAD

**Discussion by Dyuret** [community/community#62760] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

It looks like this is your first time starting a discussion in this repository!
This is a community we build together. Please be welcoming and open minded.

**Comment by Sahnesiguzel33** [comment]
aynem öyle

**Comment by mehulrajdev** [comment]
Please mark this as an answer

**Reply by mehulrajdev** [reply]
Take a look

**Reply by Sahnesiguzel33** [reply]
nereye 😉

**Reply by mehulrajdev** [reply]
please check my profile and mark some stars 
https://github.com/mehulrajdev

**Comment by Sahnesiguzel33** [comment]
> Bir göz at

**Comment by queenofcorgis** [comment]
👋 Welcome to the GitHub Community on Discussions, @Dyuret  Happy to have you here 🙂

To help keep our space clutter-free, we recommend introducing yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and checking out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776). This post will be removed within 24-48 hours to keep our community nice and tidy 🧹.
